src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -212,6 +200,38 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    private void showAbout() {
+        aboutView.getView(scrollContent.getChildren()::setAll);
+        setActiveNavItem(aboutNav);
+    }
+
+    private void showAccounting() {
```

Comment:
`showAbout` and `showAccounting` are identical. Parametrize them to `show(view, navItem)` and call 
```JAVA
if (hasIdentity()) {
    show(accountingView, accountingNav);
} else {
    show(aboutView, aboutNav);
}
```

It would be even better (for this code and for the tests) if the navItem and the view were grouped. Like nav.getView() and then the `show` method would only need the navItem to get the view, too. They belong together bidirectional anyways.